### PR TITLE
[feature] browseUI view format

### LINKED ISSUE #6: [UI] Browse View

### PR DESCRIPTION
**Pull request by juanyepesp**
New components added (just views, no functionality):
 - `SpotCard`: Creates a card for the spot
 - `HorCategoryList`:  Lists some categories, is not very responsive or scrollable atm.

Also the browse view shows usage of the components.

closes #6 

Screenshot 2024-02-25 at 10 10 08 PM

### CODE REVIEW

**Review by mar-cas3 — CHANGES_REQUESTED** [review]
Looks good, I would leave a basic ViewModel implemented for now, so just the get() method that returns a hardcoded list so later on when integrating Firebase it's a matter of changing only that code.

File: FoodBookApp/UI/Components/HorCategoryList.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//
+//  HorCategoryList.swift
+//  FoodBookApp
+//
+//  Created by Juan Diego Yepes Parra on 25/02/24.
+//
+
+import SwiftUI
+
+struct HorCategoryList: View {
```
[the remark this answers is not in the document]
Comment (juanyepesp) [reply FoodBookApp/UI/Components/HorCategoryList.swift:10]: Horizontal JSJSJS maybe I should choose another name

[the remark this answers is not in the document]
Comment (mar-cas3) [reply FoodBookApp/UI/Components/HorCategoryList.swift:10]: Maybe `HCategoryList`, like `HStack` in the SwiftUI components??

**Review by mar-cas3 — APPROVED** [review]
Looks good!

**Comment by juanyepesp** [comment]
> Looks good!

Not closing yet for merging with latest Develop changes, will ask for future revisions

**Comment by Laurarestrepo03** [comment]
Question: isn't search bar missing?

**Comment by mar-cas3** [comment]
@juanyepesp was able to resolve conflicts, feel to to merge onto develop when you're ready

**Comment by juanyepesp** [comment]
@Laurarestrepo03 
> Question: isn't search bar missing?

Yes but since it is not a feature atm I will leave it like this for now

**Pull request merged**